detection_rules/schemas/definitions.py

```diff
@@ -158,6 +158,7 @@
 Severity = Literal['low', 'medium', 'high', 'critical']
 Sha256 = NewType('Sha256', str, validate=validate.Regexp(SHA256_PATTERN))
 SubTechniqueURL = NewType('SubTechniqueURL', str, validate=validate.Regexp(SUBTECHNIQUE_URL))
+StoreType = NewType("StoreType", str, validate=validate.OneOf(["appState", "globalState"]))
```

Comment:
could change this to a literal to simplify further

`StoreType = Literal['appState', 'globalState']`